website/aggregated_questions.qmd

```diff
@@ -0,0 +1,251 @@
+
+Will a country ban polygenic screening for non-medical traits [exact traits required] by 20XX? (note: explicit ban, any country, specific trait)
+
+Will a significant [definition of significant required] legal case concerning discrimination [definition of discrimination required] based on genetic information [OR genetic engineering?] reach an international court by 20XX?
+
+Will over XX% of countries have laws in place that ban genetic modifications [scope of bans for current countries required] for non-health-related [better phrasing for "non-health-related" required] enhancements by 20XX?
+
+Will more than XX% of the world's population [OR consortium of countries OR specific country] oppose [definition of oppose required] the use of genetic editing for enhancement in public opinion polls [scope of polls required] by 20XX?
+
+Will any country legalize germline genome editing for enhancement by 20XX?
+
+Will the right to refuse genetic modifications be captured in international human rights law [which international human rights law required] by 20XX?
+
+Will genetic engineering technology for cognitive enhancements become a major topic in a national election [measurement of "major topic" required] by 20XX?
+
+Will any country mandate [definition of mandate required] genetic screening for certain genetic diseases by 20XX?
+
+Will religious organizations [scope of religious organizations needed] formally oppose genetic engineering for human enhancement in more than 25 [specific number of countries or jurisdiction scope required] countries by 20XX?
+
+Will polygenic screening for physical or cognitive traits lead to a class-action lawsuit [scope of lawsuit required] in the United States by 20XX?
+
+Will more than 30 countries [scope of countries required] introduce regulations [scope of regulations required] ensuring equal access [scope of equal access required] to genetic enhancement technologies by 20XX?
+
+Will there be a criminalization of unregulated [attempted topic of question is "black market" or "back market"] genetic modifications in over 20 countries [scope of countries required] by 20XX?
+
+Will a genetically modified individual be legally [scope of legal definition required] denied citizenship or human rights by 20XX?
+
+Will more than XX% of countries adopt ethical guidelines [scope of ethical guidelines required] prohibiting the selection of embryos based on non-health-related traits by 20XX?
+
+Will human genome editing be used as a defense in a court of law in a high-profile case [defense in country and publicity of case definitions required] by 20XX?
+
+Will there be widespread protests / riots [scope of civil disobience surrounding topic required] against the use of polygenic screening in schools or workplaces [OR other settings] by 20XX?
+
+Will any country [some scope correction probably useful here] allow parents to choose the sexual orientation of their children via genetic screening or editing by 20XX?
+
```

Comment:
specification: "explicitly permit, deregulate, or encourage genetic screening with the intent of affecting"